include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3839,7 +3839,8 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
   const auto cells = this->get_cell_ids();
 
   // 2) actually gather values
-  VectorizedArrayType out = make_vectorized_array<Number>(Number(1.));
+  VectorizedArrayType out =
+    make_vectorized_array<VectorizedArrayType>(Number(1.));
```

Comment:
I am wondering if can write `VectorizedArrayType out = Number(1.);`.

Reply:
The compiler doesn't complain, so it seems to be possible. Do you want me to change it?

Reply:
Yes. The copy assignment is a rather new feature.

Reply:
May I ask you to also fix this line:

https://github.com/dealii/dealii/blob/0c059d559e7334a1719068e6c08d5bf1e6bfc583/include/deal.II/matrix_free/fe_evaluation.h#L6060

Thanks!